R/run.R

```diff
@@ -534,48 +534,105 @@ log_table_to_run <- function(name, value, description = "", run = NULL) {
   invisible(NULL)
 }
 
-#' Plot table of run details in RStudio Viewer
+#' Plot table of run details in RStudio Viewer or browser
+#' This table does not auto-refresh. To see current values,
+#' re-run the command or click the web view link to view more
+#' details in real time.
 #' @param run The `Run` object.
 #' @export
 #' @md
 view_run_details <- function(run) {
```

Comment:
does this function return anything that should be mentioned here?

Reply:
no - no return object other than the plot itself in the viewer